web/src/containers/ApdStateProfileMedicaidOffice.test.js

```diff
@@ -1,83 +1,148 @@
-import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
 import React from 'react';
-import sinon from 'sinon';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
 
-import {
-  plain as StateProfile,
-  mapStateToProps,
-  mapDispatchToProps
-} from './ApdStateProfileMedicaidOffice';
-import { updateApd } from '../actions/apd';
+import { UPDATE_APD } from '../actions/apd';
+import { EDIT_APD } from '../actions/editApd';
+
+import StateProfile from './ApdStateProfileMedicaidOffice';
+
+const mockStore = configureStore([thunk]);
 
 describe('apd state profile, Medicaid office component', () => {
-  const props = {
-    stateProfile: {
-      medicaidDirector: {
-        name: 'name',
-        email: 'email',
-        phone: 'phone'
-      },
-      medicaidOffice: {
-        address1: 'address 1',
-        address2: 'address 2',
-        city: 'city',
-        state: 'state',
-        zip: 'zip'
+  const store = mockStore({
```

Comment:
In the `connect()` components, we export the un-connected component as `plain` and can bypass the state-to-props process altogether. In that case, we inject the props directly into the component. With redux hooks, though, there's no way to separate the state-to-props process. As a result, in order to test the component, we need to actually give it a redux store. So, here we create .a story for the component.